libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -118,6 +118,10 @@ public List<String> getUserIssuedCredentials(List<VcStoreItem> vcStoreItems) {
         return vcStoreItems.stream().map(VcStoreItem::getCredential).toList();
     }
 
+    public void deleteVcItem(String userId, String criId) {
```

Comment:
We've moved all these methods onto `VerifiableCredentialService` now, and I think this should go there too. The idea is that:
- `VerifiableCredentialService` handles all the reading/writing from the VC store
- `UserIdentityService` performs operations on the VCs themselves (e.g. checking correlation)